funnel/views/account.py

```diff
@@ -576,9 +576,7 @@ def confirm_email(email_hash, secret):
 @requires_login
 def change_password():
     if not current_auth.user.pw_hash:
-        form = PasswordResetForm()
-        form.edit_user = current_auth.user
-        del form.username
+        form = PasswordCreateForm()
```

Comment:
Doesn't this need `edit_user`? It has to check candidate against the user's fullname/username.